test/Properties.sol

```diff
@@ -105,15 +110,56 @@ abstract contract Properties is Setup {
         return true;
     }
 
-    function property_D() public returns (bool) {
-        (uint256 totalCreditWoeth,,) = oeth.creditsBalanceOfHighres(address(woeth));
-        uint256 localCreditWoeth = (woeth.oethCreditsHighres());
-        uint256 targetCreditWoeth = totalCreditWoeth - __sum_donated_credits;
-        if (!approxEqAbs(localCreditWoeth, targetCreditWoeth, t_D)) {
-            emit Log.log_named_uint("localCreditWoeth   ", localCreditWoeth);
-            emit Log.log_named_uint("totalCreditWoeth   ", totalCreditWoeth);
-            emit Log.log_named_uint("sum_donated_credits", __sum_donated_credits);
-            emit Log.log_named_uint("diff: ", delta(localCreditWoeth, targetCreditWoeth));
+    /// @dev Tested in the "afterInvariant" function
+    ///      Amount deposited/minted minus amount withdrawn/redeemed should always
+    ///      be smaller than the amount of hardAssets on the contract
+    function __property_D() internal returns (bool) {
```

Comment:
Maybe you could add a small description of what those properties are verifying under DEFINITION section?

Reply:
Good thinking. Done here: https://github.com/OriginProtocol/woeth-fuzzing-tests/pull/1/commits/fae360b8d6421111836367743e9d8e3d120ad7fd